main.go

```diff
@@ -269,9 +272,14 @@ func generate(in configInput) ([]byte, error) {
 			"":    {},
 		}
 		if in.includeDirectPathAuthority {
-			c.Authorities[c2pAuthority] = Authority{
+			authority := Authority{
 				XdsServers: generateServerConfigsFromInputs("dns:///directpath-pa.googleapis.com", in),
 			}
+			if in.includeXDSTPNameInLDS {
+				template := fmt.Sprintf("xdstp://%s/envoy.config.listener.v3.Listener/%%s", c2pAuthority)
+				authority.ClientListenerResourceNameTemplate = template
```

Comment:
Nit: Should be fine to inline this and get rid of the `template` variable.

Reply:
done. I was trying to keep the line width to 80.. but i guess there are multiple places in this file where that doesn't apply

Reply:
There is no need to keep code lines to 80 cols. That guideline is only for comment lines. 

See: https://g3doc.corp.google.com/go/g3doc/style/guide.md?cl=head#line-length and go/go-style/decisions#comment-line-length.